### PR TITLE
Disable unwrapping of inbound mails.

### PR DESCRIPTION
**Pull request by njohner**
Unwrapping of inbound mails is rather confusing, notably when there are several attachments to an inbound mail, any of which being an E-mail, it will save that attachment into Gever instead of the whole E-mail. As this behavior is not necessary anymore we simply deactivate it for all deployments.

For https://4teamwork.atlassian.net/browse/CA-2375

## Checklist
- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)
- Upgrade steps (changes in profile):
  - [ ] Make it deferrable if possible
  - [ ] Execute as much as possible conditionally

### CODE REVIEW

**Review by phgross — CHANGES_REQUESTED** [review]
Changes are looking good, but we should disable this setting also in the gever and teamraum policytemplates, because the setting is True by default.

**Comment by njohner** [comment]
thanks for the feedback. Indeed I forgot the templates!

File: opengever/policytemplates/policy_template/opengever.+package.name+/opengever/+package.name+/+adminunit.id+/profiles/default/registry.xml.bob [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <value key="is_auto_refresh_enabled">{{{setup.bumblebee_auto_refresh}}}</value>
   </records>
 
+  <records interface="ftw.mail.interfaces.IMailSettings">
```
Comment (phgross) [inline opengever/policytemplates/policy_template/opengever.+package.name+/opengever/+package.name+/+adminunit.id+/profiles/default/registry.xml.bob:12]: Just wonder, is this file also included in teamraum policies?

**Review by phgross — APPROVED** [review]
LGTM - just wonder if it works also for new teamraum policies

File: opengever/policytemplates/policy_template/opengever.+package.name+/opengever/+package.name+/+adminunit.id+/profiles/default/registry.xml.bob (hunk 1 above)
Comment (njohner) [reply opengever/policytemplates/policy_template/opengever.+package.name+/opengever/+package.name+/+adminunit.id+/profiles/default/registry.xml.bob:12]: yes.

**Pull request merged**